src/components/plots/ContinuousEmbeddingPlot.jsx

```diff
@@ -11,17 +11,39 @@ import { getCellSets } from 'redux/selectors';
 import { generateSpec, generateData } from 'utils/plotSpecs/generateEmbeddingContinuousSpec';
 import PlatformError from '../PlatformError';
 import Loader from '../Loader';
-import changeEmbeddingAxesIfNecessary from './helpers/changeEmbeddingAxesIfNecessary';
 
 const ContinuousEmbeddingPlot = (props) => {
   const {
     experimentId, config,
     plotData, truncatedPlotData,
     actions, loading, error,
-    reloadPlotData, onUpdate,
+    reloadPlotData, useReduxData, plotUuid,
```

Comment:
what is the point of `useReduxData`? What happens when it is `true` and what happens when it is `false`?

Reply:
the continuous embedding plot in data processing receives props for data, config, error and loading. The plot in plots and tables uses redux data. Thats why I added this, in case we want to use the redux data for the plot or pass props instead.

Reply:
Why does continuous embedding plot in data processing receives props for data, config, error and loading? Why aren't these things in redux? Sounds to me like an unclean approach, but I am also missing context around this decision, so maybe there is a good reason for this.

Separate question: instead of having a variable `useReduxData` can't we infer from the value of `config`, `error`, `loading`, `plotData` if we should use them or not?

Reply:
Because in configure embedding  we can plot different kinds of continuous embedding and this is for the generalisation sake.
config, error, loading, plotData might not be loaded yet and still passed and this would provide bugs. Using a flag useReduxData ensures these problems wont happen

Reply:
not related anymore, i added a wrapper component.